python-lib/ontology_tagger.py

```diff
@@ -59,8 +59,6 @@ def _list_sentences(self, row: pd.Series, text_column: AnyStr) -> List[AnyStr]:
         """Called if there is only one language specified
         Apply sentencizer and return list of sentences"""
         document = row[text_column]
-        if type(document) != str:
-            return []
         return [
             sentence.text for sentence in self.nlp_dict[self.language](document).sents
```

Comment:
Are we calling nlp(document) twice? Once to split in sentences, and another time to match?

Reply:
yes, we want to isolate the sentences first to get the infos about which tag has been found in which sentence

Reply:
I've created a CH card to explain why (story ID : 63463)

Reply:
Okay let's do that. Can you tweak the title of the story to clarify what it's about in a short sentence? For instance, "Avoid tokenizing twice to split by sentences then match keywords"